src/components/__test__/Calculator.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+
+import Calculator from '../Calculator';
+
+afterEach(() => {
+  cleanup();
+});
+
+test('test when first press 0', () => {
+  const { getByTestId } = render(<Calculator />);
+  const result = getByTestId('result');
+  const zero = getByTestId('no-zero');
+
+  fireEvent.click(zero);
+
+  expect(result.textContent).toBe('0');
+});
```

Comment:
- [ ] Please include all `calculate` test cases inside `describe` to group them together like the image below.

![image](https://user-images.githubusercontent.com/68796232/172890838-72525667-e62a-425c-ba7f-e6c5eba5916a.png)